auto_submit/test/src/service/fake_github_service.dart

```diff
@@ -98,4 +99,10 @@ class FakeGithubService implements GithubService {
   Future<bool> updateBranch(RepositorySlug slug, int number, String headSha) async {
     return true;
   }
+
+  @override
+  Future<Response> autoMergeBranch(PullRequest pullRequest) {
+    // TODO: implement autoMergeBranch
```

Comment:
Does this need to be done in this PR?

Reply:
No, the tests checking the automerge functionality are using mocks but the analyzer required all the methods to be implemented even if they are not used.